content/integrations/hashicorp-vault-keystore.md

```diff
@@ -16,260 +16,255 @@ This tutorial shows how to setup a KES server that uses [Vault's K/V engine](htt
+1m30s", max_request_size: "33554432", tls: "disabled")
+               Log Level:
+                   Mlock: supported: false, enabled: false
+           Recovery Mode: false
+                 Storage: inmem
+                 Version: Vault v1.16.1, built 2024-04-03T12:35:53Z
+             Version Sha: 6b5986790d7748100de77f7f127119c4a0f78946
 
-7. Unseal Vault Server
+==> Vault server started! Log data will stream in below:
+
+...
+
+WARNING! dev mode is enabled! In this mode, Vault runs entirely in-memory
+and starts unsealed with a single unseal key. The root token is already
+authenticated to the CLI, so you can immediately begin using Vault.
+
+You may need to set the following environment variables:
+
+    $ export VAULT_ADDR='http://127.0.0.1:8200'
+
+The unseal key and root token are displayed below in case you want to
+seal/unseal the Vault or re-authenticate.
+
+Unseal Key: g+epWYiEy2vj+7UP3c+YXRhoOjUMCC9PoihIzqSgB84=
+Root Token: hvs.O6QNQB33ksXtMxtlRKlRZL0R
+
+Development mode should NOT be used in production installations!
+```
+
+{{< /tab >}}
 
-   Once initialized, unseal the Vault using `M` out of `N` unseal key shares:
+{{< /tabs >}}
+
+{{< admonition type="note" >}}
+This starts a single-node Vault server in dev mode listening on `127.0.0.1:8200`.
+A dev server is ephemeral and is **not** meant to be run in production.
+{{< /admonition >}}
+
+### Connect Vault to the Vault CLI
+
+1. Set `VAULT_ADDR` endpoint
+
+   The Vault CLI needs to know the Vault endpoint:
    
    ```sh {.copy}
-   vault operator unseal eyW/+8ZtsgT81Cb0e8OVxzJAQP5lY7Dcamnze+JnWEDT
-   ```
-   ```sh {.copy}
-   vault operator unseal 0tZn+7QQCxphpHwTm6/dC3LpP5JGIbYl6PK8Sy79R+P2
+   export VAULT_ADDR='https://127.0.0.1:8200'
    ```
+   
+2. Set `VAULT_TOKEN`
+
+   The Vault CLI needs an authentication token to perform operations.
+   
    ```sh {.copy}
-   vault operator unseal cmhs+AUMXUuB6Lzsvgcbp3bRT6VDGQjgCBwB2xm0ANeF
+   export VAULT_TOKEN=hvs.O6QNQB33ksXtMxtlRKlRZL0R
    ```
-   
-   After submitting enough valid unseal key shares, Vault unseals and can process requests.
 
-8. Enable `K/V` Backend
+   Replace the token value to your own Vault access token, such as the `Root token` provided in the output of the `vault server -dev` command.
+
+3. Enable `K/V` Backend
 
    KES stores the secret keys at the Vault K/V backend. 
    Vault provides two [K/V engines](https://www.vaultproject.io/docs/secrets/kv), `v1` and `v2`.
    
    MinIO recommends the K/V `v1` engine.
 
+   {{< tabs "backend-vault-version" >}}
+
+   {{< tab "K/V v1" >}}
+
    The following command enables the K/V `v1` secret engine:
 
    ```sh {.copy}
    vault secrets enable -version=1 kv
    ```
    
+   {{< /tab >}}
+
+   {{< tab "K/V v2" >}}
+
    The following command enables the K/V `v2` secret engine:
    
    ```sh {.copy}
    $ vault secrets enable -version=2 kv
```

Comment:
Remove `$` so V1/V2 enable code blocks are consistent (also easier to copy/paste)
```suggestion
 vault secrets enable -version=2 kv
```